src/CommandList.java

```diff
@@ -0,0 +1,36 @@
+
+import java.util.HashMap;
+import java.util.Map;
+public class CommandList {
+    HashMap<String,Command> testCommand;
```

Comment:
Why it is not private ? Why it has type HashMap ? Why it has name 'testCommand' ?